tests/js-tests/res/Shaders/example_Twist.vsh

```diff
@@ -0,0 +1,8 @@
+// http://www.cocos2d-iphone.org
+
+attribute vec4 a_position;
+
+void main()
+{
+    gl_Position = CC_MVPMatrix * a_position;
+}
```

Comment:
Ok.
These shaders seems meaningless.
I think we should remove them, and add more meaningful shaders.